test-integ/peering_commontopo/ac5_1_test.go

```diff
@@ -0,0 +1,245 @@
+		expected := fmt.Sprintf("%s-sidecar-proxy", s.clientSID.Name)
+		services := getServicesInCluster(t, cl, &api.QueryOptions{
+			Peer: peer,
+		})
+		require.NotContains(t, services, expected, fmt.Sprintf("error: should not create proxy for service: %s", services))
+	})
+}
+
+func (s *serviceMeshDisabledSuite) testSingleQueryFailover(t *testing.T, c *api.Client, ct *commonTopo, peer string) {
+	if s.DC == "dc2" {
+		// TO-DO: currently failing
+		t.Skip()
+	}
+
+	t.Run("prepared query with single failover", func(t *testing.T) {
+		var err error
+
+		// disable server node in dc1 and relaunch topology
+		cfg := ct.Sprawl.Config()
+		require.NoError(t, disableNodeInCluster(t, cfg, ct.DC1.Name, s.serverSID.Name, true)) //dc1
+		require.NoError(t, ct.Sprawl.Relaunch(cfg))
+
+		// assert server health status
+		dc2 := ct.Sprawl.Topology().Clusters[ct.DC2.Name]
+		dc2APIClient := ct.APIClientForCluster(t, dc2)
+
+		assertServiceHealth(t, c, s.serverSID.Name, 0)            //dc1 - unhealthy
+		assertServiceHealth(t, dc2APIClient, s.serverSID.Name, 1) //dc2 - healthy
+
+		// create prepared query definition
+		def = &api.PreparedQueryDefinition{
+			Name: "ac5-prepared-query-1failover",
+			Service: api.ServiceQuery{
+				Service:     s.serverSID.Name,
+				Partition:   ConfigEntryPartition(s.serverSID.Partition),
+				OnlyPassing: true,
+				// create failover for peer in dc2 and dc3 cluster
+				Failover: api.QueryFailoverOptions{
+					Targets: []api.QueryFailoverTarget{
+						{
+							Peer: peer,
+						},
+					},
+				},
+			},
+		}
+
+		query := c.PreparedQuery()
+		def.ID, _, err = query.Create(def, nil)
+		require.NoError(t, err)
+
+		// Read registered query
+		queryDef, _, err := query.Get(def.ID, nil)
+		require.NoError(t, err)
+		require.Len(t, queryDef, 1, "expected exactly 1 prepared query")
+		require.Equal(t, 1, len(queryDef[0].Service.Failover.Targets), "expected 1 failover targets for dc2")
+		fmt.Println("PreparedQuery with failover created successfully.")
+
+		// expected outcome should show 2 failovers
+		queryResult, _, err := query.Execute(def.ID, nil)
+		require.NoError(t, err)
+		require.Equal(t, 1, queryResult.Failovers, "expected 1 failover to dc2")
+		// should failover to peer in DC2 cluster
+		require.Equal(t, ct.DC2.Datacenter, queryResult.Nodes[0].Node.Datacenter)
+		// failover to nearest cluster
+		require.Equal(t, "peer-dc2-default", queryResult.Nodes[0].Checks[0].PeerName)
+	})
+}
+
+// getServicesInCluster validates that the service(s) exist in the Consul catalog
+func getServicesInCluster(t *testing.T, c *api.Client, opts *api.QueryOptions) map[string][]string {
+	var (
+		services map[string][]string
+		err      error
+	)
+	retry.Run(t, func(r *retry.R) {
+		services, _, err = c.Catalog().Services(opts)
+		require.NoError(r, err, "error reading service data")
+		if len(services) == 0 {
+			r.Fatal("did not find service(s) in catalog")
+		}
```

Comment:
```suggestion
		require.Greater(r, len(services), 0, "did not find service(s) in catalog"))
```
^ test this, I wrote it off the cuff